DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
 ![choice card](media/choice_card.png)
 
-***Fig. A** example true\[X\] mid-roll choice card*
+_**Fig. A** example true[X] mid-roll choice card_
```

Comment:
Thanks for taking on this nit! 😞 , why is GH markdown not smart enough to handle both bold + italics at once in its preview. 

A non-issue since the rendered version looks great!